packages/reactivity/src/reactive.ts

```diff
@@ -92,6 +92,19 @@ export function shallowReadonly<T extends object>(
   )
 }
 
+// Return a reactive-copy of the original object, where only the root level
+// properties are reactive, and does NOT unwrap refs nor recursively convert
+// returned properties.
+export function shallow<T extends object>(target: T): T {
```

Comment:
Let's name this `shallowReactive` which is more explicit.

Reply:
okay, sure!